packages/sanity/src/core/form/inputs/DateInputs/base/DateTimeInput.tsx

```diff
@@ -71,7 +70,8 @@ export const DateTimeInput = forwardRef(function DateTimeInput(
   )
 
   return (
-    <LazyTextInput
+    <TextInput
```

Comment:
There seem to be some side effects cause by this. I've added more details in the thread.